### PR TITLE
Add OpenPisco

### PR DESCRIPTION
**Pull request by fbordeu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by fbordeu** [comment]
@cnrdn , @daviddanan 

 GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

**Comment by daviddanan** [comment]
I confirm i am willing to be listed here.

**Comment by cnrdn** [comment]
I confirm i am willing to be listed here.

**Comment by fbordeu** [comment]
@conda-forge/help-python, ready for review!

Recipe works on windows, linux, mac. 

Thanks for the work.

Felipe

File: recipes/OpenPisco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{% set version = "1.1.4" %}
+
+package:
+  name: openpisco-split
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/openpisco/openpisco/-/archive/{{ version }}/openpisco-{{ version }}.tar.bz2
+  #url:  /mnt/c/Users/d584808/Downloads/openpisco-1.1.4.tar.bz2
```
Comment (xhochy) [inline recipes/OpenPisco/meta.yaml:9]:
```suggestion
```

File: recipes/OpenPisco/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        - vtk
+        - libmed >=4.1.1
+        - qt =5
+    test:
+      imports:
+        - OpenPisco
+      commands:
+        - python -m OpenPisco.CLApp.OpenPiscoCL
+
+  #- name: "openpisco-extensions"
```
Comment (xhochy) [inline recipes/OpenPisco/meta.yaml:49]: Why is this still commented?

File: recipes/OpenPisco/build_core.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+set -x
+$PYTHON -m pip install --no-deps . -vv
```
Comment (xhochy) [inline recipes/OpenPisco/build_core.sh:2]: Seems like these scripts aren't necessary.

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by fbordeu** [comment]
Thanks for the feedback, currently I'm away, without a computer access. Will treat yours remarks in two weeks. Thank again for your time.

**Comment by fbordeu** [comment]
Hello @conda-forge/help-python, ready for review!

Corrected all the remarks can someone treat this PR.

Thanks to all

**Review by xhochy — APPROVED** [review]

**Pull request merged**